photon-ml/src/main/scala/com/linkedin/photon/ml/supervised/classification/LogisticRegressionModel.scala

```diff
@@ -73,12 +73,16 @@ class LogisticRegressionModel(override val coefficients: Coefficients)
 }
 
 object LogisticRegressionModel {
+
   /**
    * Create a new logistic regression model with the provided coefficients (means) and variances
    *
    * @param coefficients The feature coefficient means and variances for the model
    * @return A logistic regression model
    */
-  def create(coefficients: Coefficients): LogisticRegressionModel =
+  def create(coefficients: Coefficients): GeneralizedLinearModel =
+    new LogisticRegressionModel(coefficients)
+
+  def apply(coefficients: Coefficients): GeneralizedLinearModel =
```

Comment:
I would love to retire the "create" since "apply" is shorter, and apparently more Scala.